.drone.yml

```diff
@@ -4,7 +4,9 @@ workspace:
 
 pipeline:
   build:
-    image: golang:1.10
+    image: golang:1.14
+    environment:
+      - GO111MODULE=on
```

Comment:
If using go 1.16, this is set by default

Reply:
I don't intend to use 1.16 for now